Tasks/ChocoTask/choco.ps1

```diff
@@ -1,12 +1,21 @@
 [CmdletBinding()]
 param(
-    [Parameter()]
+    [Parameter(Mandatory=$true)]
     [string] $Package,
```

Comment:
I see that `Package` and `Version` use upper case first letter, but `ignoreChecksums` uses lower case first letter. This is nitpicking, but given the nature of this code it'd be good to converge into a single way of doing it. Reading common powershell docs on built-in commandlets leads me to believe upper case first letter is best. If you agree, please update the last parameter to read `IgnoreChecksums`.